### PR TITLE
feat(text, title, list): apply tokens

### LINKED ISSUE #5730: Consume tokens: Title, text, and List

### PR DESCRIPTION
**Pull request by srambach**
Fixes #5730 

PR right before vacation so... 🤪 

Also adds modifiers for heading styles to the title component.

### CODE REVIEW

**Comment by lboehling** [comment]
thanks sarah! this is looking good. I do notice that the top level heading "hello world" in the text component doesn't seem to be pulling in the right size. 
image

File: src/patternfly/components/Title/title.scss [hunk 1]
```diff
@@ -1,38 +1,72 @@
...
+
+  // HEADING modifiers 
+  // h1
+  --#{$title}--m-h1--LineHeight: var(--pf-t--global--font--line-height--heading);
+  --#{$title}--m-h1--FontSize: var(--pf-t--global--font--size--heading--lg);
+  --#{$title}--m-h1--FontWeight: var(--pf-t--global--font--weight--heading);
+
+  // h2
+  --#{$title}--m-h2--LineHeight: var(--pf-t--global--font--line-height--heading);
+  --#{$title}--m-h2--FontSize: var(--pf-t--global--font--size--heading-md);
```
Comment (mcoker) [inline src/patternfly/components/Title/title.scss:45]:
```suggestion
  --#{$title}--m-h2--FontSize: var(--pf-t--global--font--size--heading--md);
```

**Review by mcoker — CHANGES_REQUESTED** [review]
LGTM, just one typo in a token name.

Also may be worth considering adding these as tokens and using them in the text/content component and for the title heading level variations

```
  --pf-t--global--font--size--heading--h1: var(--pf-t--global--font--size--heading--lg);
  --pf-t--global--font--size--heading--h2: var(--pf-t--global--font--size--heading--md);
  --pf-t--global--font--size--heading--h3: var(--pf-t--global--font--size--heading--sm);
  --pf-t--global--font--size--heading--h4: var(--pf-t--global--font--size--heading--xs);
  --pf-t--global--font--size--heading--h5: var(--pf-t--global--font--size--heading--xs);
  --pf-t--global--font--size--heading--h6: var(--pf-t--global--font--size--heading--xs);
```

**Review by mcoker — APPROVED** [review]
👍👍

**Pull request merged**